resources/functions.php

```diff
@@ -37,15 +34,13 @@
 /**
  * Ensure dependencies are loaded
  */
-if (!class_exists('Roots\\Sage\\Container')) {
```

Comment:
Is this change compatible with Bedrock usage?

Reply:
yes– it is unrelated to bedrock

Reply:
Thanks but that's not my question. Actually it _is_ related to Bedrock, as we can see when git blaming the file (https://github.com/roots/sage/pull/1821).

The thing is, when using Bedrock, dependencies via composer are loaded at the Bedrock level, not at the Sage level. Then these lines will raise an error.

Reply:
It won't raise an error.